packages/viewer/src/lib/types/entities.ts

```diff
@@ -0,0 +1,8 @@
+export enum EntityType {
```

Comment:
Might be worth adding a comment with the permalink to the code that generates these values:  https://github.com/Vertexvis/rendering-worker-service/blob/fc1c864d1da4b5eb12794478dc720fa046412ee2/src/embree.isph#L163

Reply:
Updated